content/updates/2020-12/1-Convert-variant-products-into-simple-products.md

```diff
@@ -0,0 +1,17 @@
+---
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Convert variant products into simple products
+::: meta-data type="New" features="Governance, Productivity" available="late December" link-to-doc="../articles/TODO"
+
+To introduce greater flexibility when dealing with variant products, you can now easily transform them into simple products directly from the PIM interface. What a big help whenever you need to refine your family variants! Or, in the case, some of your product ranges shrink to one product instead of multiple.
```

Comment:
```suggestion
To offer greater flexibility when dealing with variant products, you can now easily transform them into simple products directly from the PIM interface. What a big help whenever you need to refine your family variants! Or, in case some of your product ranges shrink to one product instead of multiples.
```